client/components/LearnerViewDirective.js

```diff
@@ -144,10 +153,10 @@ tie.directive('learnerView', [function() {
         }
         .tie-coding-terminal, .tie-question-window {
           background-color: rgb(255, 255, 255);
-          border-color: rgb(222, 222, 222);
-          border-radius: 3px;
-          border-style: solid;
-          border-width: 1px;
+          -webkit-font-smoothing: antialiased;
+        }
+        .tie-coding-terminal.night-mode, .tie-question-window.night-mode {
```

Comment:
Why do you need a duplicate rule here that repeats the same things as the CSS rule immediately above?